definition-src/Game/LambdaHack/Content/ModeKind.hs

```diff
@@ -11,19 +11,16 @@ module Game.LambdaHack.Content.ModeKind
   ) where
 
 import Prelude ()
-
 import Game.LambdaHack.Core.Prelude
 
 import qualified Data.Text as T
-
```

Comment:
Please keep the import blocks separate, as they were. Let me know if this requires explanation.

Reply:
Thanks, I didn't know the reason. I looked it up in [kowainik.](https://kowainik.github.io/posts/2019-02-06-style-guide#imports)